.dscanner.ini

```diff
@@ -46,7 +46,7 @@ logical_precedence_check="enabled"
 ; Checks for undocumented public declarations
 undocumented_declaration_check="enabled"
 ; Checks for poor placement of function attributes
-function_attribute_check="enabled"
+function_attribute_check="disabled"
```

Comment:
Are you sure you want to disable this entirely? It seems that this check applies to other attributes (@safe, nogc etc.) as well.